src/open_inwoner/templates/pages/product/form.html

```diff
@@ -13,7 +13,7 @@
 
 {% block content %}
 
-    <h1 class="h1" id="title">
+    <h1 class="h1 utrecht-heading-1" id="title">
```

Comment:
This has both h1 and utrecht class

Reply:
I'm not sure if this template is actually used in practice anywhere but I'll change it anyway 😬